### DISCUSSION TITLE
Daily 31/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#258] [General]
Ayer terminé con los controladores, eventualmente quedó simplificado a 7 controladores distintos (que podrían simplificarse a 6 si juntase login con usuarios). El modelo quedó con las 6 clases que teníamos en el diagrama aunque libro_diario se llama ventas, después lo renombro en alguno de los dos. Y quedamos con 7 servicios que contiene la lógica de los controladores. O sea, que el usuario llama al controlador, el controlador llama al servicio y el servicio llama a los modelos que necesite. Tenemos casi 200 pruebas unitarias para verificar que todo funcione. También estuve revisando la base de datos, el login, registración y recuperación de password, la creación de artículos ya está funcionando con MySQL, durante el día voy a estar revisando si el resto funciona, espero terminar entre hoy y mañana con todo eso. Ayer instalé Wordpress en mi sitio pero aunque cambie la versión de PHP no me la toma así que hoy voy a estar hablando con soporte sobre eso, igual ya pedí un servidor nuevo para instalarlo ahí por si en el mío no se puede. Hoy también voy a empezar a armar mi CV en mi Wordpress local para entregarlo a SGC y para tenerlo listo para subirlo al sitio cuando lo tengamos arriba.

**Comment by AhumadaCeleste** [comment]
Ayer estudie durante todo el día para el examen de full stack que realice a la noche. Hoy quiero avanzar con java, y volver a hacer mi cv en wordpress, dado a que perdí el que tenia finalizado. Mañana quiero intentar funcionalidades en Java para sumar al proyecto y  luego comenzar lo antes posible y de tiempo completo al lo ultimo modulo de Python para llegar en tiempo a rendir todo lo requerido en plataforma Cisco.

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve terminando tareas del repositorio. Participe en la reunion del grupo sincrónica y estuve estudiando para la evaluación de Front,  la cual ya rendí. Hoy voy a estar revisando alguna funcionalidad mas para hacer en Java y viendo lo del diagrama de casos de uso, para poder desarrollarlo

**Comment by Brian-Almada** [comment]
Arranqué esta mañana rindiendo el examen de FS. Luego me puse al día con las clases que me perdí la semana pasada. Ahora estoy probando hacer la actividad de wordpress y luego voy a realizar tareas de ingles.
